cloud/infoblox

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/python
+#
+# Copyright 2014 "Igor Feoktistov" <ifeoktistov@yahoo.com>
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import re
+import requests
+import json
+import os
+import urllib3
+urllib3.disable_warnings()
+ca_certs = "/etc/ssl/certs/ca-certificates.crt"  # Or wherever it lives.
```

Comment:
This is not portable across distributions. For example, on RHEL 7, this file doesn't exist.